prow/config/jobs_test.go

```diff
@@ -76,8 +74,22 @@ func TestPresubmits(t *testing.T) {
 			}
 			// Check that the merge bot will run AlwaysRun jobs, otherwise it
 			// will attempt to rerun forever.
-			if job.AlwaysRun && !job.re.MatchString(testThis) {
-				t.Errorf("AlwaysRun job %s: \"%s\" does not match regex \"%v\".", job.Name, testThis, job.Trigger)
+
+			if job.AlwaysRun {
```

Comment:
If job.Always run is true then both `/test` and `@k8s-bot test this` need to trigger the job.

All jobs need to trigger on `/test JOB` and `@k8s-bot JOB test this`.